### PR TITLE
EVG-16501: clean up error message style

### PR DESCRIPTION
**Pull request by Kimchelly**
Jira: https://jira.mongodb.org/browse/EVG-16501

Clean up error message style.

### CODE REVIEW

File: bsonutil/tags.go [hunk 1]
```diff
@@ -50,10 +50,10 @@ func Tag(data interface{}, fieldName string) (string, error) {
 func MustHaveTag(data interface{}, fieldName string) string {
 	tagValue, err := Tag(data, fieldName)
 	if err != nil {
-		panic(fmt.Sprintf("error getting bson tag: %s", err.Error()))
+		panic(errors.Wrap(err, "getting BSON tag").Error())
 	}
 	if tagValue == "" {
-		panic(fmt.Sprintf("field %v cannot have an empty bson tag", fieldName))
+		panic(fmt.Sprintf("field '%s' cannot have an empty bson tag", fieldName))
```
Comment (julianedwards) [inline bsonutil/tags.go:56]: `BSON`

File: dependency_network.go [hunk 2]
```diff
@@ -107,7 +107,7 @@ func (n *dependencyNetwork) Validate() error {
 
 	for id := range dependencies {
 		if _, ok := n.network[id]; !ok {
-			catcher.Add(fmt.Errorf("dependency %s is not defined", id))
+			catcher.Add(fmt.Errorf("dependency '%s' is not defined", id))
```
Comment (julianedwards) [inline dependency_network.go:110]: `catcher.Errorf`

File: environment.go [hunk 3]
```diff
@@ -101,9 +101,9 @@ func (e *envState) Setup(q amboy.Queue, cl client.Client, session db.Session) er
 
 	e.mu.Lock()
 	defer e.mu.Unlock()
-	catcher.NewWhen(e.isSetup, "reconfiguring a queue is not supported")
+	catcher.NewWhen(e.isSetup, "reconfiguring the environment is not supported")
 
-	catcher.NewWhen(!q.Info().Started, "configuring anser environment with a non-running queue")
+	catcher.NewWhen(!q.Info().Started, "cannot set up anser environment with a non-running queue")
```
Comment (julianedwards) [inline environment.go:106]: `Anser`

**Review by julianedwards — CHANGES_REQUESTED** [review]

File: environment.go (hunk 3 above)
Comment (Kimchelly) [reply environment.go]: Done.

**Review by julianedwards — APPROVED** [review]

**Pull request merged**